pkg/trace/api/api.go

```diff
@@ -101,6 +102,15 @@ func NewHTTPReceiver(
 // Start starts doing the HTTP server and is ready to receive traces
 func (r *HTTPReceiver) Start() {
 	mux := http.NewServeMux()
+
+	runtime.SetBlockProfileRate(1) // ensure the block profile records all blocking events
```

Comment:
This one wasn't there. Do we need it? Is it "dangerous"?

Reply:
I guess it is. I thought it only sets a constant, but since you've asked I double checked, and it seems this will determine things to be recorded during runtime. We should add this back as part of another PR, hidden under a command line argument. It's not possible to get blocking profiles without this call, and as we've recently seen, we've had some serious blocking going on in our writer.

Removed for now, for the scope of this PR.